lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu'
+require_relative 'customer'
+include Menu
+include Customer
+
+class Order
+
+attr_reader :basket, :select_items, :change_quantity, :basket_count, :order_value, :total_values
+
+  def initialize
+    @order = {}
+    @min_order_value = 10
+  end
+
+  def select_items(item)
+    Menu.items {|k,v| k.match item}
+      @order.merge!(item)
+  end
+
```

Comment:
Line 17 is referring to the entire Menu class, rather than an instance of the class. You do not actually have a menu class, as Menu is currently a module.